### DISCUSSION TITLE
Dutch manual is not readable or openable.

### DISCUSSION THREAD

**Discussion by PD1LOI** [egzumer/uv-k5-firmware-custom#106] [Q&A] [answered]
[Dutch-A05-2en70enLPDenPMR-Quansheng_UV-K5_20231103-rename-2-img.docx](https://github.com/egzumer/uv-k5-firmware-custom/files/13250787/Dutch-A05-2en70enLPDenPMR-Quansheng_UV-K5_20231103-rename-2-img.docx)
Dutch manual is not readable or openable. When I open Word I get an error message.
73 PD1LOI

**Comment by ErikS-web** [comment]
Hello, that file is NOT a manual. Have a good look at the filename.

It was placed here on Wiki 
[Dutch channel list (chirp-format) for demo at 3 november 2023](https://github.com/egzumer/uv-k5-firmware-custom/wiki#dutch-channel-list-chirp-format-for-demo-at-3-november-2023)

It is a channellist. Rename extention from .docx to .img and [use Chirp](https://github.com/egzumer/uv-k5-firmware-custom/wiki/Radio-operation#basic-operation--configuration) to load it into radio.
Had to use this trick to place the list here on the Wiki for demo at the amateur clubstation

Use the [(online) Wiki](https://github.com/egzumer/uv-k5-firmware-custom/wiki#welcome-to-the-uv-k5-firmware-custom-wiki) as a guide. In this way the explanation is as up to date as possible with the latest Firmware Release.

Here's a trick to [Read this Wiki in your language](https://github.com/egzumer/uv-k5-firmware-custom/wiki#read-this-wiki-in-your-language). 
Please note: external referrals do not always work. Then go back to the original Wiki URL

**Comment by ErikS-web** [comment, marked as answer]
Kijk eens wat ik voor je heb ....

Look what I have here ....

[Egzumer017upd-Wiki2.pdf](https://github.com/egzumer/uv-k5-firmware-custom/files/13325949/Egzumer017upd-Wiki2.pdf)
